plone/app/users/browser/account.py

```diff
@@ -49,7 +49,7 @@
 
 MESSAGE_IMAGE_NOT_SUPPORTED = _(
     "message_image_not_supported",
-    "The file you selected is not supported by Pillow. " "Please choose another.",
+    "The file you selected is not supported by Pillow. Please choose another.",
```

Comment:
Unrelated to this PR, but a finding: naming Pillow here won't give most users any idea whats wrong.

Reply:
Agree, but let's not delay this small fix. Another PR is welcome.